projects/plugins/crm/includes/ZeroBSCRM.FormatHelpers.php

```diff
@@ -850,6 +850,44 @@ function zeroBSCRM_html_quoteStatusLabel($quote=array()){
 
 	}
 
+/**
+ * Return an updated HTML string, replacing quote date placeholders with correct date strings based on site settings.
+ * @param  string  $value         The value of the date variable to be replaced.
+ * @param  string  $key           The key of the date variable to be replaced.
+ * @param  string  $working_html  The HTML string to be updated.
+ * @param  boolean $custom_field Whether the key in question is a custom quote field.
+ *
+ * @return string                The updated HTML string.
+ */
+function jpcrm_process_date_variables( $value, $key, $working_html, $custom_field = false ) {
+	$placeholder_str_start = '##';
+	if ( $custom_field ) {
+		$placeholder_str_start = '##QUOTE-';
+	}
+	$numeric_date_key       = $key;
+	$string_to_numeric_date = wp_date( get_option( 'date_format' ), strtotime( $value ) );
+	$datetime_key           = $key . '_datetime_str';
+	$string_to_datetime     = wp_date( 'd F Y ' . get_option( 'time_format' ), strtotime( $value ) );
```

Comment:
Why `d F Y`?

Reply:
So, this is the part that is covered in the notes in that I made the call to use those formats for those two types of placeholders (datetime_str and date_str), as otherwise it seems pointless having those placeholders available. It does mean possible duplication with the base date placeholder depending on what date setting is selected in site settings.